test/specs/apexReplayDebugger.e2e.ts

```diff
@@ -222,6 +222,23 @@ describe('Apex Replay Debugger', async () => {
     await utilities.pause(1);
   });
 
+  step('SFDX: Launch Apex Replay Debugger with Current File - test class', async () => {
```

Comment:
@CristiCanizales there are no assertions made in this step. Is success making it through the code in this test. What if all the UI interactions succeed, but the launch actually fails?

Reply:
Right! I copied the previous step which throws no notif at all and didn't realize when launching from a test file we do get a notif that we can validate. Thanks! Test run: https://github.com/forcedotcom/salesforcedx-vscode/actions/runs/9032667289/job/24821299340